oauth2_http/java/com/google/auth/oauth2/IdentityPoolCredentials.java

```diff
@@ -269,22 +276,24 @@ private String getSubjectTokenFromMetadataServer() throws IOException {
     }
   }
 
+  /**
+   * Returns whether or not the current configuration is for Workforce Pools (which enable 3p user
+   * identities, rather than workloads).
```

Comment:
nit: Maybe you can be i bit more crisp, specifically saying that if this true - workloads will be disabled.

Reply:
Going to keep it like this for now